pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -51,7 +51,8 @@ stdenv.mkDerivation rec {
     sha256 = "15iw7982g6vc4jy1l9kk1z9sl5bm1bdbwr74y7nvwjs1nffhig7f";
   };
 
-  nativeBuildInputs = [ makeWrapper python python.pkgs.sphinx python.pkgs.sphinx_rtd_theme pkg-config flex bison meson ninja ]
+  nativeBuildInputs = with buildPackages.buildPackages; [ perl python3 python3.pkgs.sphinx python3.pkgs.sphinx_rtd_theme binutils ]
```

Comment:
`buildPackages.buildPackages` looks fishy.

Reply:
We do not need crossing binutils here, as only `strings` is used in the build, thus `buildPackages.buildPackages` is a viable choice, same for python and perl.

Reply:
maybe use depsBuildBuild instead then

Reply:
depsBuildBuild seems to be broken: https://hydra.nichi.co/build/26/nixlog/1

Reply:
fixed as of: https://hydra.nichi.co/build/40